recipes/tensorflow-lite/all/test_package/conanfile.py

```diff
@@ -1,18 +1,26 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools import build
+from conan.tools.cmake import CMake, cmake_layout
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def layout(self):
+        cmake_layout(self)
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self):
+        if not build.cross_building(self):
```

Comment:
```suggestion
        if build.can_run(self):
```
Prefer can_run. It incorporates `cross_building`, but it's managed by conan conf too. 
https://docs.conan.io/en/latest/reference/conanfile/tools/build.html#conan-tools-build-can-run